### PR TITLE
[PATCH v6] test: export: implement export in more applications

### PR DESCRIPTION
**Pull request by Rayska**

### CODE REVIEW

File: test/performance/odp_timer_accuracy.c [hunk 1]
```diff
@@ -1026,7 +1026,8 @@ static int print_stat(test_global_t *test_global)
 			return -1;
```
Comment (TuomasTaipale) [inline test/performance/odp_timer_accuracy.c:1026]: This fix patch could be separated to its own PR, so we get it merged more quickly.

Comment (Rayska) [reply test/performance/odp_timer_accuracy.c]: Created a new PR for minor fixes: https://github.com/OpenDataPlane/odp/pull/2087

File: test/performance/odp_pool_perf.c [hunk 2]
```diff
@@ -645,6 +648,31 @@ static void print_stat(test_global_t *global)
 	       (1000.0 * frees_ave) / nsec_ave);
 	printf("  events per sec:       %.3f M\n\n",
 	       (1000.0 * events_ave) / nsec_ave);
+
+	printf("TOTAL events per sec:       %.3f M\n\n",
```
Comment (MatiasElo) [inline test/performance/odp_pool_perf.c:652]: Couple extra spaces. Print would look nicer if the value would be aligned with the previous ones.

File: test/performance/odp_sched_latency.c [hunk 3]
```diff
@@ -255,7 +259,7 @@ static int enqueue_events(int prio, int num_queues, int num_events,
  *
  * @param globals  Test shared data
  */
-static void print_results(test_globals_t *globals)
+static int output_results(test_globals_t *globals)
```
Comment (MatiasElo) [inline test/performance/odp_sched_latency.c:262]: All applications could use the same function name for outputting the results. I'd perhaps prefer this one (`output_results()`).

File: test/performance/odp_stash_perf.c [hunk 4]
```diff
@@ -496,7 +531,8 @@ int main(int argc, char **argv)
 	/* Wait workers to exit */
 	odph_thread_join(global->thread_tbl, global->options.num_cpu);
 
-	print_stat(global);
+	if (output_stats(global))
+		return -1;
```
Comment (MatiasElo) [inline test/performance/odp_stash_perf.c:535]: For uniformity `exit(EXIT_FAILURE);`

**Review by MatiasElo — APPROVED** [review]
